Makefile

```diff
@@ -44,19 +45,12 @@ help:
 	@echo " * 'uninstall'      - Remove installed binaries from system locations"
 	@echo " * 'version'        - Print current cri-containerd release version"
 
-.PHONY: check-gopath
-
-check-gopath:
-ifndef GOPATH
-	$(error GOPATH is not set)
-endif
-
 verify: lint gofmt boiler
 
 version:
 	@echo $(VERSION)
 
-lint: check-gopath
```

Comment:
The problem is that `check-gopath` is a PHONY,, target depends on it will run every time.
I want to avoid rebuild, if the binaries are built or source code is not changed, so I have to remove this.